pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -463,7 +464,8 @@ func getTPMapMatchingIncomingAffinityAntiAffinity(pod *v1.Pod, nodeInfoMap map[s
 		nodeInfo := nodeInfoMap[allNodeNames[i]]
 		node := nodeInfo.Node()
 		if node == nil {
-			catchError(fmt.Errorf("nodeInfo.Node is nil"))
+			catchError(fmt.Errorf("node not found"))
```

Comment:
Would you mind updating to:

```suggestion
			catchError("node %q not found", node.Name)
```